fastlane/appstoreres/metadata/source/description.txt

```diff
@@ -9,4 +9,6 @@ Get notifications about store activity – including new orders and product revi
 
 WooCommerce is the most customizable eCommerce platform for building your online business. Built on WordPress, it is integrated with the world’s best content management tools. From coffee subscriptions to Spanish lessons to gym memberships to complex enterprise-level eCommerce – visit WooCommerce.com/start to launch a new store and ship your idea.
 
-Requirements: WooCommerce v3.5+, the Jetpack plugin.
\ No newline at end of file
+Requirements: WooCommerce v3.5+, the Jetpack plugin.
+
+View the Privacy Notice for California Users at https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.
```

Comment:
I'll leave the review for this file to Platform 9

Reply:
Yep this is fine, we'll just need to add it to the translation `.pot` file. I can open a PR for that tomorrow separately.

Reply:
Thanks @jkmassel - I swear I looked for that file and couldn't find it. I thought maybe it was no longer necessary 😛

Reply:
Yeah! I think this is enough. It will be picked up by the script during the next code freeze and sent for translation.

Reply:
Ah ok perfect – I always did it by hand just in case for metadata! 😄